http-netty/src/main/java/io/micronaut/http/netty/channel/EventLoopGroupFactory.java

```diff
@@ -35,6 +38,7 @@
  * @author graemerocher
  * @since 1.2.0
  */
+@NextMajorVersion("This doesn't create event loops anymore. Rename to e.g. NettyTransport")
 public interface EventLoopGroupFactory {
```

Comment:
Frankly I would just deprecate the whole class with `forRemoval` rather than leave it and make it do something it doesn't do anymore

Reply:
we still need all the implementations, so this class still has a purpose. it's just that the abstraction level has changed–it's not event loops being created anymore, it's IoHandlerFactories (and channel classes)